automatic/codelite/tools/chocolateyInstall.ps1

```diff
@@ -5,7 +5,6 @@ $toolsPath = Split-Path $MyInvocation.MyCommand.Definition
 $packageArgs = @{
   packageName    = 'codelite'
   fileType       = 'exe'
-  file           = "$toolsPath\codelite_x32.exe"
```

Comment:
Please set `file` to an empty string instead of removing it entirely (partly due to backwards compatibility, and to explicitly say that 32bit is not supported).

```
  file = ''
```

Reply:
Addressed in e10de2025c411f583603e2a7f52ea5e76ee67814.